pkg/pb/kt/ktarget.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package kolide.launcher;
+
+option go_package = "kt";
+
+service KTarget {
+  rpc GetTargets(GetTargetsRequest) returns (GetTargetsResponse) {}
+}
+
+message GetTargetsRequest {
+  string node_key = 1;
+}
+
+message GetTargetsResponse {
+  repeated Target targets = 1;
+}
+
+message Target {
+  string id = 1;
```

Comment:
This drops the type enum?

Reply:
I feel like we're talking in circles. What happens when I want to add SQL support?

Reply:
proto is always forward compatible, so at that point you'd just add extra fields you need to the `Target` message (including the type enum)

 ```proto
message Target {
  string id = 1;
  TargetType target_type = 2;
  string sephs_sql_field = 3;
}

enum TargetType {
   <...>
}
```

Reply:
I think the value of TargetType changes how string is handled.

Reply:
no issue there - it's future functionality, so you can handle it however you like